portal/src/graphql/adminapi/AuditLogList.tsx

```diff
@@ -141,7 +143,7 @@ const AuditLogList: React.FC<AuditLogListProps> = function AuditLogList(props) {
       let href: string | null = null;
       switch (column?.key) {
         case "activityType":
-          href = `./${item.id}/details/`;
+          href = `${item.id}/details${searchParam}`;
```

Comment:
It is unnecessary to pass the query around. For example, we did not pass the query to the user details page link. But we still can restore the search state when we pop the user details page.